app/src/main/java/cm/aptoide/pt/home/HomePresenter.java

```diff
@@ -30,10 +37,13 @@
   private final AdMapper adMapper;
   private final AptoideAccountManager accountManager;
   private final HomeAnalytics homeAnalytics;
+  private final HighlightedAdExperiment highlightedAdExperiment;
```

Comment:
This should probably be in the home interactor (home.java) and not the presenter.